internal/assets/.arikawa-boilerplate.example.yaml

```diff
@@ -15,3 +10,24 @@ logger:
 
     # copies is the number of log files to retain.
     copies: 3
+
+# http defines the parameters for the HTTP server.
+http:
```

Comment:
Just to check, this can also be set via environment variables right?

Reply:
correct